### PR TITLE
Added visual Bubble Sort 

### PR DESCRIPTION
**Pull request by akshitadixit**
## Related Issue
  - Attempts to solve #217 

### Describe the changes you've made
* Added bubbleSort.html for structure
* Added bubbleSort.css for formatting
* Added bubbleSort.js for the functionality
* Added bubble sort in the cards list in script.js so it shows up on home page
* Added a gif for the card
### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

### CODE REVIEW

**Comment by urvashi-code1255** [comment]
@akshitadixit Resolve the merge conflicts

**Comment by akshitadixit** [comment]
Right away

**Review by urvashi-code1255 — COMMENTED** [review]
@akshitadixit I can't see your project on the website. Please have a look!

**Comment by akshitadixit** [comment]
> @akshitadixit I can't see your project on the website. Please have a look!

That is because it isn't merged yet.

**Comment by urvashi-code1255** [comment]
> That is because it isn't merged yet.

Hey @akshitadixit I mean the website I am seeing locally at your PR is not showing the project

File: assets/js/bubbleSort.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+function sorting() {
+    genArray();
+    delay1 = 0;
+    bubbleSort();
+}
+
+function bubbleSort() {
+    // left the algorithm implementation for the other assignee
```
Comment (991rajat) [inline assets/js/bubbleSort.js:46]: Try to implement the Algorithm

Comment (akshitadixit) [reply assets/js/bubbleSort.js:46]: I'm ready but @urvashi-code1255 asked me to distribute work and I did mine.

Comment (991rajat) [reply assets/js/bubbleSort.js:46]: You can go ahead
After that Squash the commits.
> Let me know if you need any help.

Comment (akshitadixit) [reply assets/js/bubbleSort.js:46]: Sure.

**Comment by Vishal-raj-1** [comment]
@akshitadixit Resolve conflicts then I will merge your PR

**Pull request merged**